openpype/hosts/houdini/plugins/publish/extract_opengl.py

```diff
@@ -46,6 +47,12 @@ def process(self, instance):
             "camera_name": instance.data.get("review_camera")
         }
 
+        colorspace = ropnode.evalParm("ociocolorspace")
+        # inject colorspace data
+        self.set_representation_colorspace(
+            representation, instance.context,
+            colorspace=colorspace
+        )
```

Comment:
How accurate is this? Is the user technically stil capable of doing exports (e.g. if houdini is not color managed) **without** OCIO colorspaces? If so, should this be like this?
```python
if ropnode.evalParm("colorcorrect") == 2:  # OpenColorIO enabled
    colorspace = ropnode.evalParm("ociocolorspace")
    # etc..
```
?

That would also avoid issues where if someone had entered a colorspace value in `ociocolorspace` but then swapped to just use `None` or `LUT and Gamma` color correction where the `ociocolorspace` parm still has the string value that was set - giving confusion here as to why it's still using that data?

Reply:
good point! I missed that!